Raise CMake minimum version requirement to squash warnings
Don't redefine KDE4WORKSPACE_WEATHERION_LIBRARY as it causes linking failure

--- CMakeLists.txt.orig	2013-03-23 08:31:22 UTC
+++ CMakeLists.txt
@@ -1,5 +1,5 @@
 PROJECT(plasma-applet-yawp)
-cmake_minimum_required(VERSION 2.6)
+cmake_minimum_required(VERSION 2.8.9)
 
 #if you don't want the full compiler output, remove the following line
 SET(CMAKE_VERBOSE_MAKEFILE ON)
@@ -40,10 +40,6 @@ else ( ${KDE_VERSION} VERSION_LESS "4.1.
 	set( PLASMA_LIBS ${KDE4_PLASMA_LIBS} )
 endif ( ${KDE_VERSION} VERSION_LESS "4.1.90" )
 
-if ( ${KDE_VERSION} VERSION_GREATER "4.2.9" )
-	set( KDE4WORKSPACE_WEATHERION_LIBRARY weather_ion)
-endif ( ${KDE_VERSION} VERSION_GREATER "4.2.9" )
-
 FIND_PACKAGE(Qt4 REQUIRED QtNetwork QtGui QtCore QtXml QtSvg QtDbus)
 
 
